packages/core/services/signers/BitcoinCashLedgerDevice.js

```diff
@@ -15,9 +14,8 @@ export const API = axios.create({
   baseURL: BASE_URL,
 })
 
-export default class BitcoinCashLedgerDevice extends EventEmitter {
+export default class BitcoinCashLedgerDevice {
```

Comment:
Why you are removed eventEmitter?

Reply:
Because, as I see, eventEmitter ain't using at all. There are no events at BitcoinCashLedgerDevice